tests/test_ev_operator.py

```diff
@@ -15,6 +20,22 @@
 from eko.thresholds import ThresholdsAtlas
 
 
+def test_full_labels_dict():
+    for mode in ["S_qq", "S_qg", "S_gq", "S_gg"]:
+        assert full_labels_dict[mode] < 4
```

Comment:
Test explicitly the encoding, one by one. At least we get a reference that is enforcing the thing.